main/webapp/modules/core/scripts/reconciliation/standard-service-panel.html

```diff
@@ -1,11 +1,15 @@
 <div class="recon-dialog-service-panel recon-dialog-standard-service-panel">
   <div class="grid-layout layout-normal layout-full grid-layout-for-ui"><table>
     <tr>
+      <td colspan="2" style="text-align: right;">
+        <a href="" target="_blank" bind="documentationLink"><span bind="or_proc_accessDocumentation"></span>
+        <span style="display: inline-block"></span></a>
+      </td>
       <td colspan="2" style="text-align: right;">
         <a href="" target="_blank" bind="rawServiceLink"><span bind="or_proc_access"></span>
-        <span class="ui-icon ui-icon-link" style="display: inline-block"></span></a>
```

Comment:
It looks like you removed the existing "Access service API" link but you kept its `<td>`, and you added another `<td>` for the documentation link. This means that you are still deforming the table by creating a new column that is empty in the other rows.
I would just reuse the existing `<td>` element instead.

Reply:
I removed the TD element please have a look!
Thankyou